### ISSUE TITLE
TinyMCE Settings 

### ISSUE THREAD

**Issue by sarath-sasikumar** [DefinitelyTyped/DefinitelyTyped#16267]
In the tinyMCE.init 
The settings in the type does not have the option of 
paste_data_images: boolean
As such I am unable to give this configuration

```
import * as tinymce from "tinymce";

tinymce.init({
paste_data_images: true
});
```
Is it possible to give these specific configurations in some other way,
like `advlist_number_styles` ,`paste-data-images`
At the moment it is not possible to provide such configurations.

**Comment by andy-ms** [comment]
You could make a PR adding that property.

**Comment by sarath-sasikumar** [comment]
Have made a PR #19235

**Comment by andy-ms** [comment]
Fixed by #19235.
